### PR TITLE
[SoapClient] Fixes __getLastRequest and __getLastResponse

### PR DESCRIPTION
**Pull request by laland**
This fixes ```VCR/Utils/SoapClient```'s ```__getLastRequest()``` and ```__getLastResponse()``` stuff, but i dont have a mind how to deal with __getLastRequestHeaders and __getLastResponseHeaders, and looks like i don't need it 'cause it just returns http headers but not SOAP headers. I can parse SOAP headers out of request/response body, so i give up on fixing __getLastRequestHeaders and __getLastResponseHeaders for now.

### CODE REVIEW

**Comment by laland** [comment]
adri?

**Comment by adri** [comment]
Was sorry @laland. This has been lying around for too long :/

File: src/VCR/Util/SoapClient.php [hunk 1]
```diff
@@ -17,6 +17,10 @@ class SoapClient extends \SoapClient
 
     protected $options = array();
 
+    protected $response;
```
Comment (adri) [inline src/VCR/Util/SoapClient.php]: Docblocks would be great for this, so the type is clear.

File: src/VCR/Util/SoapClient.php [hunk 2]
```diff
@@ -45,9 +51,21 @@ public function __doRequest($request, $location, $action, $version, $one_way = 0
             $response = $this->realDoRequest($request, $location, $action, $version, $one_way);
         }
 
+        $this->response = $response;
+
         return $one_way ? null : $response;
     }
 
+    public function __getLastRequest()
```
Comment (adri) [inline src/VCR/Util/SoapClient.php]: Good solution. Also docblocks would be great :)

**Comment by adri** [comment]
Ok docblocks and it's good to merge

**Comment by laland** [comment]
Hi @adri, is it ok now?

**Comment by adri** [comment]
Perfect thanks!

**Pull request merged**